os/fs/dirent/fs_closedir.c

```diff
@@ -156,6 +157,7 @@ int closedir(FAR DIR *dirp)
 
 	/* Then release the container */
 
+	memset(idir, 0, sizeof(struct fs_dirent_s));
```

Comment:
Hi @jeongarmy , I have a question.
Why 'memset' required before 'free' ?

free function doesn't delete somethings in that region. => What is means?

Reply:
@JunhwanPark 
The 'free' fucntion just unsets ALLOC mask which was set when malloc.
It means that somethings we wrote are still remained.